.github/workflows/publish.yml

```diff
@@ -29,6 +29,8 @@ jobs:
     name: NPM Package
     needs: sanity-check
     runs-on: ubuntu-latest
+    permissions:
+      id-token: write
```

Comment:
Seems to me that this permission should be documented, as is not clear what it does. Is missing in the documentation of the `JS-DevTools/npm-publish` action.

Reply:
It's in npm's documentation: https://docs.npmjs.com/generating-provenance-statements#publishing-packages-with-provenance-via-github-actions

Reply:
Ah, OK then...